lib/llm/AnthropicClient.ts

```diff
@@ -79,7 +87,7 @@ export class AnthropicClient extends LLMClient {
             },
           },
         });
-        return cachedResponse;
+        return cachedResponse as T;
```

Comment:
can we do something like `const cacheOptions: AnthropicTransformedResponse | undefined = this.cache.get...` to avoid `as` forced typecast